ui/components/component-library/modal-body/modal-body.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
This storybook file follows the latest format from the storybook website https://storybook.js.org/docs/get-started/whats-a-story